sdks/cpp/src/agones/sdk.cc

```diff
@@ -26,8 +28,33 @@ struct SDK::SDKImpl {
   std::unique_ptr<grpc::ClientContext> health_context_;
 };
 
+std::string port() {
```

Comment:
The C++ Style guide says that this function name should start with a capital letter and likely have some verb in it. So I'd suggest `GetPort` or `LookupPort` or something like that.